### PR TITLE
Add initial module transform.py with delta method

### PR DESCRIPTION
**Pull request by kels271828**

### CODE REVIEW

File: tests/test_transforms.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+"""Tests for transforms.py module."""
```
Comment (jiaweih) [inline tests/test_transforms.py:1]: Would you add tests for expected results of some instances ?

Comment (kels271828) [reply tests/test_transforms.py:1]: Good idea, I can add something

File: src/distrx/transforms.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+    Notes
+    -----
+    The delta method expands a function of a random variable about its
+    mean with a one-step Taylor approximation and then takes the
+    variance.
+
+    """
+    mu, sigma = np.array(mu), np.array(sigma)
+    _check_input(mu, sigma, transform)
```
[the remark this answers is not in the document]
Comment (kels271828) [reply src/distrx/transforms.py]: Added checks to both transform_data and delta_method.

File: tests/test_transforms.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+VALS = [0.1]*2
+
+
+@pytest.mark.parametrize('transform', TRANSFORM_LIST)
+def test_method_name_valid(transform):
+    """Raise ValueError for invalue `method`."""
+    with pytest.raises(ValueError):
+        transform_data(VALS, VALS, transform, method='dummy')
+
+
```
[the remark this answers is not in the document]
Comment (kels271828) [reply tests/test_transforms.py:19]: Done

File: tests/test_transforms.py [hunk 4]
```diff
@@ -18,29 +18,29 @@ def test_method_name_valid(transform):
 
 
 @pytest.mark.parametrize('function', FUNCTION_LIST)
-def test_input_len_match(function):
+@pytest.mark.parametrize('transform', TRANSFORM_LIST)
```
Comment (jiaweih) [inline tests/test_transforms.py:36]: Looks nice!

**Pull request merged**